signer/src/stacks/events.rs

```diff
@@ -783,6 +868,61 @@ mod tests {
         };
     }
 
+    #[test]
+    fn test_key_rotation_event() {
+        let new_keys: Vec<PublicKey> = vec![
+            fake::Faker.fake_with_rng(&mut OsRng),
+            fake::Faker.fake_with_rng(&mut OsRng),
+            fake::Faker.fake_with_rng(&mut OsRng),
+        ];
```

Comment:
Oh generating public keys is a little more annoying, I'd generate a `secp256k1::Keypair` and get a public key through that.

Reply:
Great suggestion thanks!